Dockerfile

```diff
@@ -1,8 +1,12 @@
-FROM tensorflow/tensorflow:2.5.0
-WORKDIR /app
-COPY requirements.txt /app
-RUN mkdir -p /app/data && pip install -r /app/requirements.txt
+FROM tensorflow/tensorflow:2.9.1
+ENV PASTEY_WORKERS=2
+ENV PASTEY_THREADS=4
+ENV PASTEY_LISTEN_PORT=5000
+RUN pip install gunicorn
```

Comment:
Were you experiencing some problem which made you install gunicorn separately, vs. in the requirements file?

Reply:
So the way the Dockerfile was structured it was rebuilding a lot of the dependencies any time a change was made. So I tried to improve the layers of the Dockerfile. 

That said, I think for the exact same reason the gunicorn install should be the second to last line, so that someone wishing to change it to another deployment method only has to rebuild the last two layers. I will push the change.

Reply:
Hmm. Actually the gunicorn change shouldn't be in this pull request as its a separate pull request.  Let me fix that.